src/sdl/sdl_joystick.c

```diff
@@ -83,6 +82,7 @@ void _al_sdl_joystick_event(SDL_Event *e)
    }
    else if (e->type == SDL_JOYDEVICEADDED || e->type == SDL_JOYDEVICEREMOVED) {
       event.joystick.type = ALLEGRO_EVENT_JOYSTICK_CONFIGURATION;
+      sdl_reconfigure_joysticks();
```

Comment:
looking at `ex_joystick_hotplugging` it seems the expectation is that programs should decide themselves to reconfigure. If so, should this line be removed (and any program that needs to just will call `al_reconfigure_joysticks` on this event themselves?)

Reply:
Ah, so does `ex_joystick_events`. OK, this line probably doesn't belong here.